### ISSUE TITLE
Would be so fucking great if you add the heroquest dice ;) 

### ISSUE THREAD

**Issue by altair984** [BernhardPosselt/foundryvtt-special-dice-roller#7]

**Comment by BernhardPosselt** [comment]
If you can come up with the free and legal images for each die face and explain me the rules, why not.

**Comment by altair984** [comment]
rules:

Each turn there is an attack vs. defense roll and The number of dice varies.

Hero attack roll vs defense Monster

when a hero attacks each skull is 1 damage but in the monster's defense roll each skull shield the monster draws is 1 less damage.
example:

the hero attacks with 5 dice and rolls 4 skulls against 2 monster's defense dice and the monster rolls 2 skull shields

4 skulls against 2 skull shields = 2 damage
10 skulls against 1 shield skulls = 9 damage
3 skulls against 3 shields skulls = 0 damage

when Monster Attack vs defense Hero

when a monster attacks each skull is 1 damage but in the hero's defense roll each lion's shield that the hero draws is 1 damage less.
example:

the monster attacks with 5 dice and rolls 4 skulls against 2 hero defense dice and the hero rolls 2 lion shields.

4 skulls against 2 lion shields = 2 damage
10 skulls against 1 lion shield = 9 damage
3 skulls against 3 lion shields = 0 damage

In other words, each skull in attack rolls is one of damage. Each lion shield protects 1 of damage to only the heroes in its defense roll and each skull shield protects 1 of damage to each monster in its defense roll.

On attack rolls, shields are ignored.
In the defense rolls, the skulls you draw are ignored as the different shields (the skull shield doesn't protect the hero, neither the lion shield protects the monster).

Im so sorry for my bad english, i really try to be so clear. 

![hqdice](https://user-images.githubusercontent.com/52784850/81869594-96fb0680-9574-11ea-9e4f-4543e355ed7a.png)

**Comment by altair984** [comment]
u are the king :)
